playground/terraform/infrastructure/gke/variables.tf

```diff
@@ -54,4 +54,9 @@ variable "min_count" {
 variable "max_count" {
   description = "Max cluster node count"
   default     = 6
-}
\ No newline at end of file
+}
+variable "control_plane_cidr" {
+  description = "CIDR block for GKE controlplane"
+  default     = "10.129.0.0/28"
```

Comment:
Same question, where is this range coming from?

Reply:
Arbitrary range. Private GKE cluster requires to define a CIDR for the controlplane.